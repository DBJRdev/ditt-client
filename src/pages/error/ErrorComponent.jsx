import React from 'react';

const ErrorComponent = () => (
  <div>
    <h1>Error</h1>
    <p>Page not found</p>
  </div>
);

export default ErrorComponent;
