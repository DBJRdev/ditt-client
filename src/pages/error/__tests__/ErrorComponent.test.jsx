import React from 'react';
import { shallow } from 'enzyme';
import ErrorComponent from '../ErrorComponent';

describe('rendering', () => {
  it('renders correctly', () => {
    const tree = shallow(<ErrorComponent />);
    expect(tree).toMatchSnapshot();
  });
});
