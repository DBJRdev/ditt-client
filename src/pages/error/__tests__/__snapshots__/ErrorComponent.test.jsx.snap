// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `
<div>
  <h1>
    Error
  </h1>
  <p>
    Page not found
  </p>
</div>
`;
